metrics-operator/api/v1alpha1/keptnmetric_types.go

```diff
@@ -20,7 +20,6 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
```

Comment:
Please remove the NOTE comment at all places as well

Reply:
Alright.

Reply:
Done

Reply:
I don't seem to figure out why this check is failing.
`Markdown checks / markdown-link-check (pull_request) `

Reply:
it's not related to this PR, we can ignore it, I already approved it